api/models/camera.py

```diff
@@ -61,3 +61,16 @@ class Config:
                 'contour_roi': [[88, 58], [90, 284], [279, 284], [281, 58]]
             }
         }
+
+
+class InOutBoundaries(BaseModel):
+    in_line: Tuple[Tuple[int, int], Tuple[int, int]]
```

Comment:
You will also need the direction of movement for each line. Not sure about the best way to store it

Reply:
Wasn't it from `in` to `out`?

Reply:
Nope. Each line has a direction. Only if a person is moving in that direction it will be counted. In/Out could possibly be the same line but will have opposite directions